### PR TITLE
Adding truss for sdxl turbo

### PR DESCRIPTION
**Pull request by htrivedi99**

### CODE REVIEW

File: stable-diffusion/sdxl-turbo/config.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  cover_image_url: https://cdn.baseten.co/production/static/sd.png
+  tags:
+  - image-generation
+model_name: SDXL Turbo
+python_version: py310
+requirements:
+  - torch==2.0.1
+  - transformers==4.35.2
+  - diffusers==0.23.1
+  - scipy==1.11.3
```
Comment (varunshenoy) [inline stable-diffusion/sdxl-turbo/config.yaml:16]: where's the scipy req coming from?

File: stable-diffusion/sdxl-turbo/config.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  - hf_transfer==0.1.4
```
Comment (varunshenoy) [inline stable-diffusion/sdxl-turbo/config.yaml:17]: did you run into issues if `hf_transfer` wasn't included?

File: stable-diffusion/sdxl-turbo/model/model.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    def convert_to_b64(self, image: Image) -> str:
+        buffered = BytesIO()
+        image.save(buffered, format="JPEG")
+        img_b64 = base64.b64encode(buffered.getvalue()).decode("utf-8")
+        return img_b64
+
+    def predict(self, model_input: Dict) -> Dict:
+        prompt = model_input.get("prompt")
+        output_image = self.model(
+            prompt=prompt, num_inference_steps=1, guidance_scale=0.0
```
Comment (varunshenoy) [inline stable-diffusion/sdxl-turbo/model/model.py:29]: let's parameterize num steps and guidance_scale via model_input

**Review by varunshenoy — COMMENTED** [review]
Looks good! Just a few comments/questions.

File: stable-diffusion/sdxl-turbo/model/model.py (hunk 3 above)
Comment (varunshenoy) [reply stable-diffusion/sdxl-turbo/model/model.py:29]: guidance_scale = 0.0 is a bit weird. Curious why this is recommended.

File: stable-diffusion/sdxl-turbo/config.yaml (hunk 2 above)
Comment (htrivedi99) [reply stable-diffusion/sdxl-turbo/config.yaml:17]: Yes, I ran into issues when this dependency wasn't included.

File: stable-diffusion/sdxl-turbo/model/model.py (hunk 3 above)
Comment (htrivedi99) [reply stable-diffusion/sdxl-turbo/model/model.py:29]: Based on the blog post I read, num_inference_steps is supposed to be one and guidance_scale is not used by this model so it's set to 0. The user should not be able to interact with these values. I can make them constants and move them to the top of the file if you like.

File: stable-diffusion/sdxl-turbo/config.yaml (hunk 1 above)
Comment (htrivedi99) [reply stable-diffusion/sdxl-turbo/config.yaml:16]: Looks like scipy is not needed. Good catch!

**Review by varunshenoy — APPROVED** [review]
lgtm!

**Pull request merged**